schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -84,6 +87,55 @@
       ],
       "description": "Microsoft.Sql/servers"
     },
+    "servers_administrators": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "activeDirectory"
```

Comment:
Yes this is the only valid name. @fearthecowboy sounds like autorest should not be generating this